package.json

```diff
@@ -23,8 +23,8 @@
         "test-all": "node --enable-source-maps node_modules/@definitelytyped/dtslint-runner/ --path .",
         "clean": "node scripts/remove-empty.js",
         "clean-node-modules": "node scripts/clean-node-modules.js",
-        "test": "node --enable-source-maps node_modules/@definitelytyped/dtslint/ types",
-        "lint": "node --enable-source-maps node_modules/@definitelytyped/dtslint/ types",
+        "test": "node --enable-source-maps node_modules/@definitelytyped/dtslint/ types/relationship.js",
+        "lint": "node --enable-source-maps node_modules/@definitelytyped/dtslint/ types/relationship.js",
```

Comment:
This command is supposed to be used for testing any package. You can use it as `pnpm run tst relationship.js`. Reverting this specific change in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67793

Reply:
Got it. Thank you